sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/OrganizationDetails.java

```diff
@@ -23,7 +23,7 @@
      * Details of the organization administrator.
      */
     @JsonProperty(value = "admin_details")
-    private List<Administrator> adminDetails;
+    private List<AdministratorContact> adminDetails;
```

Comment:
should `adminDetails` be `administratorContacts` instead ?

Reply:
internal class, should map to REST API as it is.